roles/irods/tasks/icat.yml

```diff
@@ -1,29 +1,167 @@
-#installing the iRODS ICAT
+# installing the iRODS ICAT
+- name: Create {{ irods_service_account }} group
+  group:
+    name: "{{ irods_service_account }}"
+    state: present
+  become: true
+
+- name: Creating {{ irods_service_account }} user
+  user:
+    name: "{{ irods_service_account }}"
+    groups: "{{ irods_service_account }},admin"
+  become: true
+
 - name: Installing the iRODS Package(s)
-  become: True
-  yum: 
-    name: "{{ item }}"
+  yum:
+    name: '{{ item }}'
     state: present
   loop:
-  - irods-server
-  - irods-database-plugin-postgres
+    - irods-server
+    - irods-database-plugin-postgres
+    - irods-resource-plugin-s3-4.2.10.0-1
```

Comment:
Is it necessary to hard-code the version number for this S3 plugin?

Reply:
Correct. Tested without - works. Removed in new commit.